presto-iceberg/src/main/java/com/facebook/presto/iceberg/IcebergAbstractMetadata.java

```diff
@@ -632,6 +633,36 @@ public void rollback()
         // TODO: cleanup open transaction
     }
 
+    @Override
+    public void dropBranch(ConnectorSession session, ConnectorTableHandle tableHandle, Optional<String> branchName)
+    {
+        String branch = branchName.get().replace("'", "");
```

Comment:
Why are we doing `.replace()` here?

Reply:
Here the branch String would come something like `'<branch-name>'` so stripping that before accessing

Reply:
Is there a reason it's quoted? I think the value passed here should not include the quotes. You might need to edit the parser grammar to strip any quotes

Reply:
This is something open for discussion. As of now I made this quote based on the syntax in the [issue](https://github.com/prestodb/presto/issues/22028). We can discuss if we want to have grammar like other ALTER TABLE operations https://prestodb.io/docs/current/sql/alter-table.html#examples ?

Reply:
Sounds good to me. Updated the PR